deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
can't we just put the following on their own lines, or does this allow some dependencies - where the second command only runs if the first is complete.